README.md

```diff
@@ -1,150 +1,170 @@
+                                                            // on it's own, beneath the "Install App" header. If set to 'inline', the
+                                                            // app name will be displayed on a single line like "Install MyApp"
+                                                            // Optional. Default 'standalone'
+     appIconUrl: 'apple-touch-icon.png',                    // App icon link (square, at least 40 x 40 pixels).
                                                             // Required.
      assetUrl: 'https://cdn.jsdelivr.net/gh/philfung/add-to-homescreen@2.0/dist/assets/img/',  // Link to directory of library image assets.
 
      maxModalDisplayCount: -1                               // If set, the modal will only show this many times.
                                                             // Optional. Default: -1 (no limit).  (Debugging: Use this.clearModalDisplayCount() to reset the count)
    });
-         
+
     ret = window.AddToHomeScreenInstance.show('en');        // show "add-to-homescreen" instructions to user, or do nothing if already added to homescreen
                                                             // The only argument is the language to show the messages in (currently only 'da', 'de', 'en', 'es', 'fr', 'he', 'it', 'pt' and 'ru' are available).
    });
    </script>
    </body>
    ```
-   
-Here's an [example implementation](https://github.com/philfung/add-to-homescreen/blob/main/index.html). 
+
+Here's an [example implementation](https://github.com/philfung/add-to-homescreen/blob/main/index.html).
 
 #### Special Case: calling the UI later
- 2-alternate. if you're calling the UI NOT onload, but sometime after (for example, in an onclick() handler for an "Install App" button), then
+
+2-alternate. if you're calling the UI NOT onload, but sometime after (for example, in an onclick() handler for an "Install App" button), then
 you should still create your the instance onload, but call your UI later on the instance variable with .show()):
 
-  `index.html`
-   ```javascript
-   <script>
-   document.addEventListener('DOMContentLoaded', function () { 
+`index.html`
 
-      window.AddToHomeScreenInstance = new window.AddToHomeScreen({...
-   }));
+```javascript
+<script>
+document.addEventListener('DOMContentLoaded', function () {
 
-   document.getElementById('my_install_app_button').addEventListener('click', function () {
-      window.AddToHomeScreenInstance.show('en');
-   });
- </script>
-   </body>
-   ```
+   window.AddToHomeScreenInstance = window.AddToHomeScreen({...
+}));
+
+document.getElementById('my_install_app_button').addEventListener('click', function () {
+   window.AddToHomeScreenInstance.show('en');
+});
+</script>
+</body>
+```
 
 This is because some handlers must be created onload.
 
 ### Usage (If you're making changes to library)
+
 1. Make changes
 2. Test locally:
 
 Start local server
+
 ```
 npm run install
```

Comment:
this should be "npm install"

Reply:
/shaggy 

:-)

Reply:
Courtesy of my little Chrome extension https://gitme.me !!